src/providers/seventv/SeventvEmotes.cpp

```diff
@@ -131,114 +139,44 @@ boost::optional<EmotePtr> SeventvEmotes::emote(const EmoteName &name) const
 
 void SeventvEmotes::loadEmotes()
 {
-    qCDebug(chatterinoSeventv) << "Loading 7TV Emotes";
-
-    QJsonObject payload, variables;
-
-    QString query = R"(
-        query loadGlobalEmotes($query: String!, $globalState: String, $page: Int, $limit: Int, $pageSize: Int) {
-        search_emotes(query: $query, globalState: $globalState, page: $page, limit: $limit, pageSize: $pageSize) {
-            id
-            name
-            provider
-            provider_id
-            visibility
-            mime
-            owner {
-                id
-                display_name
-                login
-                twitch_id
-            }
-        }
-    })";
-
-    variables.insert("query", QString());
-    variables.insert("globalState", "only");
-    variables.insert("page", 1);  // TODO(zneix): Add support for pagination
-    variables.insert("limit", maxPageSize);
-    variables.insert("pageSize", maxPageSize);
+    qCDebug(chatterinoSeventv) << "[7TVEmotes] Loading 7TV Global Emotes";
 
-    payload.insert("query", query.replace(whitespaceRegex, " "));
-    payload.insert("variables", variables);
+    QString url("https://api.7tv.app/v2/emotes/global");
 
-    NetworkRequest(apiUrlGQL, NetworkRequestType::Post)
+    NetworkRequest(url)
         .timeout(30000)
-        .header("Content-Type", "application/json")
-        .payload(QJsonDocument(payload).toJson(QJsonDocument::Compact))
-        .onSuccess([this](NetworkResult result) -> Outcome {
-            QJsonArray parsedEmotes = result.parseJson()
-                                          .value("data")
-                                          .toObject()
-                                          .value("search_emotes")
-                                          .toArray();
-            qCDebug(chatterinoSeventv)
-                << "7TV Global Emotes" << parsedEmotes.size();
-
-            auto pair = parseGlobalEmotes(parsedEmotes, *this->global_.get());
+        .onSuccess([this](auto result) -> Outcome {
+            auto emotes = this->emotes();
+            auto pair = parseGlobalEmotes(result.parseJsonArray(), *emotes);
             if (pair.first)
                 this->global_.set(
                     std::make_shared<EmoteMap>(std::move(pair.second)));
             return pair.first;
+            return Failure;
```

Comment:
Unreachable return?

Reply:
Fix this, but seems I didn't push before I logged off for the night. This should be removed.